src/conspiracies/preprocessing/preprocessor.py

```diff
@@ -47,9 +47,11 @@ def _validate_content(
             else:
                 yield doc
 
-    def preprocess_docs(self, input_path: str, output_path: str):
+    def preprocess_docs(self, input_path: str, output_path: str, n_docs: int = -1):
```

Comment:
I would use None instead of -1, to make it clear that is shouldn't be interpreted as a meaningful number.

Reply:
Sure, that seems to be the pythonic way. This is my Java programming habits seeping in. :smile:

Reply:
thought as much (it also happens in e.g. sklearn so it is not completely unheard of in python)